managed/services/supervisord/supervisord.go

```diff
@@ -372,6 +374,9 @@ func (s *Service) UpdateLog(offset uint32) ([]string, uint32, error) {
 	for {
 		line, err := reader.ReadString('\n')
 		if err == nil {
+			if newOffset > s.gRPCMessageMaxSize {
+				return lines, newOffset, errors.WithStack(err)
```

Comment:
`err` here is `nil`

Reply:
BTW, we just return error an that's it? How it solves the problem?

Reply:
@artemgavrilov where do we return an error? I think idea to return part of the log that fits grpc message size?
But it probably should be after this line: `lines = append` as otherwise `lines`is empty.

Looks like log api has pagination (request log by offset) and this forces this pagination.

Reply:
@artemgavrilov , @denisok fixed

Reply:
but `lines` have nothing ?

Reply:
it does, it contains everything we collect during previous loops. it can be empty only in one case: if the length of one line is higher than max limit, which is almost impossible 
if we put this if after `lines = append` the size of lines message might become bigger than max limit.

Reply:
and there was error that I was comparing newOffset with max limit instead of diff between newOffset and offset

Reply:
yep, I see it now